src/app/campaign-groups.service.ts

```diff
@@ -442,7 +442,7 @@ export class CampaignGroupsService {
     );
 
     // For an unknown/invalid beneficiary, show 'other' symbol.
-    return matchingItems[0].icon || faPlus;
```

Comment:
this line specifically is quite similar to the one fixed before. Would have thrown a TypeError if matchingItems[0] was undefined.